pep-0215.txt

```diff
@@ -69,7 +69,7 @@ Examples
 ========
 
 Here is an example of an interactive session exhibiting the
-expected behaviour of this feature::
+expected behaviour of this feature. ::
```

Comment:
Ah, I see, thanks for clearing that up. 

A dot instead of a comma seems like a good replacement. The comma version, at least for me, not so much. I think it would out of place before a code block (while `:` seems natural).